homeassistant/components/forked_daapd/media_player.py

```diff
@@ -827,12 +827,13 @@ def _use_pipe_control(self):
             return self._source[:-7]
         return ""
 
-    async def _pipe_call(self, pipe_name, base_function_name):
-        if self._pipe_control_api.get(pipe_name):
-            return await getattr(
-                self._pipe_control_api[pipe_name],
+    async def _pipe_call(self, pipe_name, base_function_name) -> None:
+        if pipe := self._pipe_control_api.get(pipe_name):
+            await getattr(
+                pipe,
                 PIPE_FUNCTION_MAP[pipe_name][base_function_name],
             )()
+            return
```

Comment:
Small refactor here: use a walrus and always discard the return value, as it was never used